event/src/args.rs

```diff
@@ -84,141 +107,66 @@ impl<'a, S: Signer> EventArgs<'a, S> {
     }
 }
 
-#[derive(Serialize)]
-struct UnsignedEventHeader {
-    controllers: Vec<String>,
-    #[serde(rename = "model")]
-    parent: Bytes,
-    sep: &'static str,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    unique: Option<Vec<u32>>,
-}
-
-/// An unsigned event, which can be used to create a events
-#[derive(Serialize)]
-pub struct UnsignedEvent<T: Serialize> {
-    #[serde(skip_serializing_if = "Option::is_none")]
-    data: Option<T>,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    header: Option<UnsignedEventHeader>,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    prev: Option<Cid>,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    id: Option<Cid>,
-}
-
-fn create_unique() -> Result<Vec<u32>> {
+fn create_unique() -> Result<Bytes> {
     let mut rng = rand::thread_rng();
-    let mut unique = [0u32; 12];
+    //TODO is u8; 12 OK?
```

Comment:
how do we plan to address this TODO?  Can you link a ticket number where we will figure this out?

Reply:
What can the unique value be in an event? Previously it was treated as u32s but I was told its always opaque bytes. If we can get a clear answer to this question then we can remove the TODO.

Reply:
You can see both ways this can be generated here: https://github.com/ceramicnetwork/js-ceramic/blob/0b6ff7c825ef915cd24aea4591904f710cae253f/packages/stream-model-instance/src/model-instance-document.ts#L458-L466

Reply:
Ok looks like we can always assume its a sequence of bytes. Will remove the TODO, we really need to specify these details in the spec.